Cargo.toml

```diff
@@ -59,3 +59,6 @@ async-trait = "0.1"
 
 aquatic_udp_protocol = "0.2"
 uuid = { version = "1", features = ["v4"] }
+
+[dev-dependencies]
+mockall = "0.11.3"
```

Comment:
Maybe we can just specify the major version, 0.11, here?

Reply:
Done.